.github/workflows/example-typescript-check-and-lint.yml

```diff
@@ -16,12 +16,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        working-directory:
-          [
-            apps/paper-example,
+        working-directory: [
+            apps/common-app,
             apps/fabric-example,
-            apps/web-example,
+            apps/macos-example,
+            # We don't check for Next.
```

Comment:
Maybe we should explain why we don't check for `Next` instead of just leaving it commented out.